include/zfp.h

```diff
@@ -145,6 +145,7 @@ typedef struct {
   int minexp;         /* minimum floating point bit plane number to store */
   bitstream* stream;  /* compressed bit stream */
   zfp_execution exec; /* execution policy and parameters */
+  unsigned long long * offset_table; /* table with the chunk offsets in bits */
```

Comment:
Why not size_t?

Reply:
Agreed, I will change this to be a fixed width type. uint64_t is good for now.